adl/language/scanner.ts

```diff
@@ -684,7 +695,9 @@ export class Scanner {
    */
   positionFromOffset(offset: number): Position {
     let position = { line: 0, character: 0, offset: 0 };
-    if (offset < 0 || offset >this.#length) {
+
+    // eslint-disable-next-line
```

Comment:
```suggestion
    // eslint-disable-next-line keyword-spacing
```

(And sheesh, that's an annoying eslint bug. 🙁)